backend/scripts/location_differ.py

```diff
@@ -162,15 +173,22 @@ def location_differ(
                             # Easy case, we just add this machine to server_dict
                             if len(device_dict[this_link]) > 1:
```

Comment:
Move check down to server case bc it cant happen here

Reply:
Done